### ISSUE TITLE
Error when adding comments between @apply

### ISSUE THREAD

**Issue by egoist** [tailwindcss/tailwindcss#817]
`foo.css`:

```css
.btn {
  @apply flex w-20 py-1 justify-center items-center;
  /* it works without this line */
  @apply border border-gray-200 rounded;
}
```

<img width="1464" alt="屏幕快照 2019-04-02 下午1 43 48" src="https://user-images.githubusercontent.com/8784712/55378844-5ea19000-554d-11e9-87cb-f638449d937a.png">

Seems like [`perfectionist`](https://github.com/ben-eb/perfectionist/) is unmaintained and it uses an outdated version of PostCSS.

---

- tailwindcss: `1.0.0-beta.4`
- postcss: `7.0.14`

**Comment by adamwathan** [comment]
Fixed in #847! Thanks.

**Comment by rightaway** [comment]
@adamwathan Does it also fix error for comments that are after like `@apply flex;  /* comment */`?

Is this fix only available in 1.0 release?

**Comment by adamwathan** [comment]
This probably does fix that yeah, and right now it's only in 1.0 but after I release 1.0 I can see about backporting it.
